Integrated Gameplay Systems (Prototype)/Assets/Scripts/Main/GameManager.cs

```diff
@@ -8,36 +8,52 @@ public class GameManager : MonoBehaviour
     public SceneBuilder sceneBuilder;
     public InputManager inputManager;
 
+    private List<BasicObject> newObjects = new List<BasicObject>();
     private List<BasicObject> objects = new List<BasicObject>();
+    private List<BasicObject> oldObjects = new List<BasicObject>();
     private Dictionary<string, object> tagList = new Dictionary<string, object>();
 
     private void Awake()
     {
         prefabLibrary.PrepareLibrary();
-        scriptableObjectLibrary.PrepareLibrary();
+        scriptableObjectLibrary?.PrepareLibrary();
```

Comment:
Goede toevoeging, moeten we dit niet ook bij de prefabLibrary doen? Want het lijkt me ook mogelijk om geen prefabs te hebben.